tools/cli/datalab.py

```diff
@@ -170,6 +192,12 @@ def run():
                 dest='zone',
                 default=None,
                 help=_ZONE_HELP)
+        if command_config['require-email']:
+            subcommand_parser.add_argument(
+                '--email',
```

Comment:
This is just a temporary solution I am using to test the changes to the Docker image.

This change isn't ready for review yet.